src/index.tsx

```diff
@@ -3,10 +3,17 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { FirebaseCharacterStore } from './models/Character';
+
+const characterStore = new FirebaseCharacterStore(
+  {
+    devServer: process.env.NODE_ENV === 'production' ? null : 'http://localhost:8080/characters',
```

Comment:
This is nice and neatly sidesteps the massive inconsistency I added by running an express app in Dev

Reply:
glad you like :) I nearly went full hog and implemented a local and firebase implementation but decided it wasn't a useful use of time

might be sweet to see what it would take to just use firebase in local dev, saves on extending a server fake over time I guess